src/score.c

```diff
@@ -336,6 +356,24 @@ int score()
 	    points = 0;
     }
 
+    /* HA2OS mods */
+    // only continent list allowed
+    if (continentlist_only == 1) {
+	is_cont_mult = continent_found();
+	if (is_cont_mult == 1) {
+	    // if we are on DX continent
+	    if (continent == mycontinent) {
```

Comment:
That is no string comparison above. It only tests if the addresses (not the content) of  both strings are the same.